lib/src/layer/polyline_layer.dart

```diff
@@ -55,33 +55,35 @@ class Polyline {
 }
 
 class PolylineLayer extends StatelessWidget {
-  /// List of polylines to draw.
   final List<Polyline> polylines;
-
   final bool polylineCulling;
 
   const PolylineLayer({
     super.key,
     this.polylines = const [],
     this.polylineCulling = false,
-    @Deprecated('No longer needed and will be removed.')
+    @Deprecated(
+      'No longer has an effect, and no alternative is available. '
+      'This option overcomplicated the situation, and is now decided automatically internally. '
+      'This feature is removed (and this option deprecated) since v5.',
+    )
```

Comment:
I think this should just be removed as it has had a default value, and has no effect now anyways. It is already "broken". However I decided to look at the internal behavior of polyline culling and i don't see any culling actually happening internally.

Reply:
Fixed in 59d3cc1.